build.gradle

```diff
@@ -87,7 +87,7 @@ def useDownloadedLibs = !System.getenv('CI') && ext.libsGitSha != null
 
 if (useDownloadedLibs) {
     task downloadAndroidLibs(type: Download) {
-        src "https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/project.application-services.application-services.build.libs.android.${rootProject.ext.libsGitSha}/artifacts/public/target.tar.gz"
+        src "https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/project.application-services.application-services.build.libs.android.${rootProject.ext.libsGitSha}/artifacts/public/android.tar.gz"
```

Comment:
Note: our toolchain scripts in the standard caching routes publish under `{android,linux,macos,win}.tar.gz` whereas the old indexes expected `target.tar.gz` so I had to adjust `build.gradle` for that. I can alternatively point our scripts to the old file format but that would change the sha of the scripts which is also taken into consideration when computing the hash digest (based on https://hg.mozilla.org/ci/taskgraph/file/tip/src/taskgraph/transforms/job/toolchain.py#l64) and new caches would be added.